packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1270,6 +1270,18 @@ abstract class ResidentRunner {
     }
   }
 
+  void maybeCallDevToolsUriServiceExtension() {
+    _devToolsLauncher ??= DevtoolsLauncher.instance;
+    if (_devToolsLauncher.activeDevToolsServer != null) {
+      unawaited(invokeFlutterExtensionRpcRawOnFirstIsolate(
```

Comment:
I think this needs to catch any RPC errors thrown

Reply:
added a try catch